packages/suite/src/services/google.ts

```diff
@@ -1,15 +1,22 @@
 /* eslint no-throw-literal: 0 */
+/* eslint @typescript-eslint/camelcase: 0 */
```

Comment:
File-wide exceptions should be reduced as much as possible.

Reply:
There are like 6 occurences of this. I dunno.. But the other one (no-throw-literal) may be now removed!